app/controllers/project/settings/donations/payments.js

```diff
@@ -55,20 +55,20 @@ export default Controller.extend({
       RSVP.hash(promises)
           .then(({ tokenData, stripeConnectAccount }) => this._addBankAccount(tokenData, stripeConnectAccount))
           .catch((response) => this._handleError(response))
-          .finally(() => set(this, 'isBusy', false));
+          .finally(() => this._endAction());
     },
 
     onVerificationDocumentSubmitted(stripeFileUploadId) {
-      set(this, 'isBusy', true);
+      this._startAction();
 
       get(this, 'stripeConnectAccount')
         .then((account) => this._assignIdentityVerificationDocument(account, stripeFileUploadId))
         .catch((response) => this._handleError(response))
-        .finally(() => set(this, 'isBusy', false));
+        .finally(() => this._endAction());
     },
```

Comment:
I personally prefer grouping  by behavior or order of operations, but have nothing against alpha ordering